### ISSUE TITLE
ACM \Description

### ISSUE THREAD

**Issue by sportdeath** [arXiv/html_feedback#4566]
### Description

[My paper](https://arxiv.org/abs/2508.04889) includes figure descriptions for accessibility [per ACM guidelines](https://authors.acm.org/proceedings/production-information/describing-figures). In the HTML, the description text renders inline not as [alt text](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt) and displays the command "\Description" in red as an error. For examples see here:

<img width="722" height="692" alt="Image" src="https://github.com/user-attachments/assets/81281d3a-6d26-4df7-ab85-5a8b39f0c66d" />

### (Required) What is this issue most closely related to? Select one.

Figures

### Internal issue ID

08748df3-9705-4c4a-b366-c92e344ddf2a

### Paper URL

https://arxiv.org/html/2508.04889v1

### Browser

Firefox/141.0

### Device Type

Desktop

**Comment by dginev** [comment]
This will land soon, pending on https://github.com/brucemiller/LaTeXML/pull/2591

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
